app.py

```diff
@@ -59,11 +59,12 @@ def main():
         )
         collateral_token = streamlit.selectbox(
             label="Select collateral token:",
-            options=list(src.settings.COLLATERAL_TO_DEBT_TOKENS.keys()),
+            options=list(src.settings.COLLATERAL_TOKENS),
             index=0,
         )
         
-        debt_token_options = src.settings.COLLATERAL_TO_DEBT_TOKENS[collateral_token]
+        debt_token_options = src.settings.DEBT_TOKENS
+        
         debt_token = streamlit.selectbox(
```

Comment:
In case when the user chooses the same collateral and debt tokens (I know it's not possible at this moment, but it will be possible in the future), could you pls add a warning message saying that the user has chosen the same tokens? 🙏🏼